rubocop.yml

```diff
@@ -383,6 +383,7 @@ Naming/InclusiveLanguage:
         - !ruby/regexp '/origin[ \/]master/' # Legacy default git branch name
         - 'mastercard'
         - 'webmaster'
+        - 'remaster'
```

Comment:
Should we just allow any unbroken word that contains "master"? Something like `/\wmaster\w/`.